openquake/hazardlib/gsim/coeffs_table.py

```diff
@@ -182,6 +184,30 @@ def non_sa_coeffs(self):
         return {imt: self._coeffs[imt] for imt in self._coeffs
                 if imt.string[:2] != 'SA'}
 
+    def get_coeffs(self, coeff_list):
```

Comment:
I would expect a ``get_coeffs(self, kind, coeff_names)`` where kind can be "SA" or "EA" and then you can use ``imt.startswith(kind)``